### DISCUSSION TITLE
Middle Quarter!

### DISCUSSION THREAD

**Discussion by Jimmbo** [rxhanson/RectanglePro-Community#614] [Ideas]
"Fourth" is the perfect window size for text docs. "Third" leaves too much white space to the right of the text.

I usually want those docs in center screen. So I yearn for a "Middle Fourth" option!

I realize "Middle Fourth" is intrinsically illogical, yet it's also easily possible!

I'm a writer, and can, if you like, try to brew up a more logical term to use. But I feel like your readers would understand (and appreciate!) a "Middle Fourth" option, despite the cognitive pain it might give a dev....

**Comment by rxhanson** [comment]
![image](https://github.com/user-attachments/assets/af552146-67d2-4c00-9ffc-3730ba11daf7)
